python/spglib/spglib.py

```diff
@@ -97,6 +107,82 @@ class SpglibError:
 spglib_error = SpglibError()
 
 
+@dataclasses.dataclass(eq=True, frozen=True)
+class DictInterface(Mapping[str, "Any"], ABC):
```

Comment:
```suggestion
class DictInterface(Mapping[str, "Any"]):
```

Remove ABC, it should not be abstract.